src/runner.ts

```diff
@@ -18,10 +17,20 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-const scan = require('../../build/src/index').scan;
 
-const options = process.argv.length > 2 ? process.argv.slice(2) : [];
+import { scan } from './scan';
+import { program } from 'commander';
+const version = require('../package.json').version;
```

Comment:
NP: We can use ES6 imports AFAIK
```suggestion
import { version } from '../package.json';
```